packages/suite/src/support/messages.ts

```diff
@@ -7729,38 +7729,47 @@ export default defineMessages({
     TR_SESSION_PHASE_11_MESSAGE: {
         id: 'TR_SESSION_PHASE_11_MESSAGE',
         defaultMessage: 'Looking for a round',
+        description: '29 symbols max',
```

Comment:
FYI, this description is currently not synced to crowdin so translators can't see them...

Reply:
oh...............

Can it be though? It's the perfect place to mention things like that.

Reply:
Sure, somebody have to implement it in the sync script. But still it's good to use it and we can sync it later.